### PR TITLE
chore: update add-to-project workflow

### PR DESCRIPTION
**Pull request by nagash77**
update add-to-project workflow to add issues created by github-actions[bot] to triage board.  Also added support for issues from the internal cypress-support repo to be added to the triage board.

- Closes 

### PR Tasks

- [ n/a ] Have tests been added/updated?
- [ n/a ] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [ n/a ] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?

### CODE REVIEW

File: .github/workflows/triage_add_to_project.yml [hunk 1]
```diff
@@ -37,7 +37,7 @@ jobs:
           echo 'IS_COLLABORATOR='$(jq -r '.data.repository.collaborators.totalCount' collaborators.json) >> $GITHUB_ENV
       - uses: actions/add-to-project@v0.4.1
         # only add issues/prs from outside contributors to the project
-        if: ${{ env.IS_COLLABORATOR == 0 }}
+        if: ${{ env.IS_COLLABORATOR == 0 || github.event.repository.name == "cypress-support-internal" || github.event.pull_request.user.login == "github-actions[bot]" || github.event.issue.user.login == "github-actions[bot]"}}
```
Comment (mschile) [inline .github/workflows/triage_add_to_project.yml]:
```suggestion
        if: ${{ env.IS_COLLABORATOR == 0 || github.event.repository.name == "cypress-support-internal" || github.event.pull_request.user.login == "github-actions[bot]" || github.event.issue.user.login == "github-actions[bot]" }}
```

**Review by mjhenkes — APPROVED** [review]

File: .github/workflows/triage_add_to_project.yml [hunk 2]
```diff
@@ -37,7 +37,7 @@ jobs:
           echo 'IS_COLLABORATOR='$(jq -r '.data.repository.collaborators.totalCount' collaborators.json) >> $GITHUB_ENV
       - uses: actions/add-to-project@v0.4.1
         # only add issues/prs from outside contributors to the project
-        if: ${{ env.IS_COLLABORATOR == 0 }}
+        if: ${{ env.IS_COLLABORATOR == 0 || github.event.repository.name == "cypress-support-internal" || github.event.pull_request.user.login == "github-actions[bot]" || github.event.issue.user.login == "github-actions[bot]" }}
```
Comment (emilyrohrbough) [inline .github/workflows/triage_add_to_project.yml:40]: It feels weird to be exposing internal/private repos in this way. Is this workflow better suited to live else-where at this point in time?

Comment (nagash77) [reply .github/workflows/triage_add_to_project.yml:40]: What way would you suggest?  What is your concern exactly?

Comment (emilyrohrbough) [reply .github/workflows/triage_add_to_project.yml:40]: discussed in post scrum. change look fine 👍🏻

**Review by emilyrohrbough — APPROVED** [review]

**Review by mschile — APPROVED** [review]

**Comment by MikeMcC399** [comment]
This PR is causing workflow failures:

```
[Invalid workflow file: .github/workflows/add-issue-to-triage-board.yml#L11](https://github.com/cypress-io/cypress-example-kitchensink/actions/runs/4665758619/workflow)
The workflow is not valid. In .github/workflows/add-issue-to-triage-board.yml (Line: 11, Col: 11): Error from called workflow cypress-io/cypress/.github/workflows/triage_add_to_project.yml@develop (Line: 40, Col: 13): Unexpected symbol: '"cypress-support-internal"'. Located at position 61 within expression: env.IS_COLLABORATOR == 0 || github.event.repository.name == "cypress-support-internal" || github.event.pull_request.user.login == "github-actions[bot]" || github.event.issue.user.login == "github-actions[bot]" cypress-io/cypress/.github/workflows/triage_add_to_project.yml@develop (Line: 40, Col: 13): Unexpected symbol: '${{'. Located at position 1 within expression: ${{ env.IS_COLLABORATOR == 0 || github.event.repository.name == "cypress-support-internal" || github.event.pull_request.user.login == "github-actions[bot]" || github.event.issue.user.login == "github-actions[bot]" }}
```

Edit: It looks like this was fixed now in https://github.com/cypress-io/cypress/commit/d6bd4ec9e76131ce9cc72c51af621150e6bd70c5.

**Pull request merged**